db/migrate/20200716141930_create_language_attributes.rb

```diff
@@ -0,0 +1,14 @@
+class CreateLanguageAttributes < ActiveRecord::Migration[5.2]
+  def change
+    create_table :language_attributes do |t|
+      t.integer :language_id, null: false
+      t.integer :resource_id, null: false
+      t.string :key, null: false
+      t.string :value, null: false
+      t.boolean :is_translatable, default: false
+      t.references :resource, null: false
```

Comment:
No `t.timestamps`? Sadly, these are missing from many existing tables, but I thought it was best practice to add them everywhere?

Reply:
Good call, thanks, I had copied this from schema.rb's language table